www_src/pages/project/project.jsx

```diff
@@ -331,21 +392,18 @@ var Project = React.createClass({
               page,
               selected: page.id === this.state.selectedEl,
               transform: this.cartesian.getTransform(page.coords),
-              onClick: this.selectPage(page)
+              onClick: isPlayOnly ? this.zoomToPage.bind(this, page.coords) : this.selectPage.bind(this, page)
```

Comment:
i would put this logic in the this.selectPage handler -- you could rename it to `onPageClick` or something it that makes more sense